gitmanager.py

```diff
@@ -73,7 +73,7 @@ def comment_on_thread(cls, thread_id, body):
         return response.json()
 
     @classmethod
-    def get_pull_request(cls, pr_id, payload):
+    def get_pull_request(cls, pr_id, payload=""):
```

Comment:
Is this change necessary and useful?

Reply:
I guess this actually fixes a bug which had gone unnoticed?

Reply:
Yeah, a payload isn't really required when sending a GET request broadly speaking. If you're supplying a payload, generally you'd be wanting to do so in a POST request. In this case, we just need to perform a GET to get a PR, so a blank payload is sufficient. This argument, really, could be completely removed.